Frontend/UserTech/src/app/components/reservation-list/reservation-list.component.spec.ts

```diff
@@ -25,8 +25,8 @@ describe('ReservationListComponent', () => {
     bikes: [
       {
         id: '1',
-        reservedAt: new Date().toLocaleString(),
-        reservedTill: new Date(Date.now() + 10000000000).toLocaleString()
+        reservedAt: new Date().toString(),
+        reservedTill: new Date(Date.now() + 10000000000).toString()
```

Comment:
Musiałem to zmienić, bo na moim komputerze wywalało błędy przy próbie konwersji tego z powrotem na Date